tests/post/steps/test_liveness.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from pytest_bdd import scenario, given, when, then, parsers
+
+# Scenarios
+@scenario('../features/pods_alive.feature', 'get pods')
+def test_pods(host):
+    pass
+
+# Given
+
+@given(parsers.parse("The '{service_name}' service is up"))
+def check_service(host, service_name):
+   assert host.service(service_name).is_running
+
+
+# Then
+@then(parsers.parse("the '{resource}' list should not be empty in the '{namespace}' namespace"))
+def check_resource_list(host, resource, namespace):
+    cmd = ("sudo kubectl --kubeconfig=/etc/kubernetes/admin.conf"
```

Comment:
Please use https://testinfra.readthedocs.io/en/latest/modules.html#sudo instead of invoking `sudo`.